examples/serverless/main.tf

```
@@ -127,3 +127,46 @@ resource "aws_rds_cluster_parameter_group" "example_mysql" {
   description = "${local.name}-aurora-mysql-cluster-parameter-group"
   tags        = local.tags
 }
+
+################################################################################
+# RDS Aurora Module - Serverless V2
+################################################################################
+
+data "aws_rds_engine_version" "test" {
```

```suggestion
data "aws_rds_engine_version" "postgresql" {
```

Is a bit more descriptive than `test`, and it is also a bit more forward-compatible when MySQL is being added as ServerlessV2 in the future.

---

updated